core/cli/src/informant.rs

```diff
@@ -154,7 +148,7 @@ pub fn start<C>(service: &Service<C>, exit: ::exit_future::Exit, handle: TaskExe
 
 fn speed(best_number: u64, last_number: Option<u64>) -> String {
 	let speed = match last_number {
-		Some(num) => (best_number.saturating_sub(num) * 10_000 / TIMER_INTERVAL_MS) as f64,
+		Some(num) => (best_number.saturating_sub(num) * 10_000 / STATUS_INTERVAL_MS) as f64,
```

Comment:
I guess we don't have to be super strict about reporting, but note that there is no guarantee that this is actually being run at this specific interval. If we want to be precise it might be better to store the time and compute the actual interval.

Reply:
Yes that's a good point, I have opened an issue for it(I think this current change does not make the situation worse compared to what we done before with the futures interval).

Reply:
Actually I've attempted to improve things somewhat, please take a look at the last commit...